includes/options/class-amp-options-manager.php

```diff
@@ -623,6 +624,41 @@ public static function show_response_cache_disabled_notice() {
 		);
 	}
 
+
+	/**
+	 * Outputs an admin notice if the site is not served over HTTPS.
+	 *
+	 * @since 1.3
+	 *
+	 * @return void
+	 */
+	public static function insecure_connection_notice() {
+		// is_ssl() only tells us whether the admin backend uses HTTPS here, so we add a few more sanity checks.
+		$uses_ssl = (
+			is_ssl()
+			&&
+			( strpos( get_bloginfo( 'wpurl' ), 'https' ) !== 0 )
+			&&
+			( strpos( get_bloginfo( 'url' ), 'https' ) !== 0 )
+		);
+
+		if ( ! $uses_ssl && 'toplevel_page_' . self::OPTION_NAME === get_current_screen()->id ) {
+			printf(
+				'<div class="notice notice-warning"><p>%s</p></div>',
+				sprintf(
+					/* translators: %s: "Why should I use HTTPS" support URL */
+					wp_kses(
+						__( 'Your site is not being fully served over a secure connection (using HTTPS).<br>' .
```

Comment:
String concatenation doesn't work for translatable strings. Given that there are line breaks, perhaps there should be three `__()` calls here.

Reply:
Erm, I already changed that... I think ... 🤔

Reply:
Ah, I changed it and then went on to add escaping to the other calls as well, but finally decided against doing that in the same PR and scrapped the changes.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3219/commits/d842fa8083805504c12bcab01524e6e35261ef56